src/utils/getPrayerDataFromFilename.ts

```diff
@@ -10,9 +10,11 @@ type PrayerDataPropsType = Readonly<{
 
 const PRAYERS = {
   wigilia: { name: 'wigilia', order: 1 },
-  jutrznia: { name: 'jutrznia', order: 2 },
-  nieszpory: { name: 'nieszpory', order: 3 },
-  kompleta: { name: 'kompleta', order: 4 },
+  kompleta_1: { name: 'kompleta I', order: 2 },
```

Comment:
I have no idea what to do with Saturday complines. They should appear after the eve, not on Saturday column as we have right now

Reply:
Requested by Puchcia